common/workunit/workunit.cpp

```diff
@@ -6166,9 +6150,8 @@ void CLocalWorkUnit::releaseFile(const char *fileName)
     }
 }
 
-void CLocalWorkUnit::clearGraphProgress()
+void CLocalWorkUnit::clearGraphProgress() const
```

Comment:
It's const so it can be part of the CConstWorkUnit interface and not require that someone lock the workunit before calling it.